src/node.rs

```diff
@@ -15,8 +15,22 @@ pub trait NodeExt<'a> {
 
 impl<'a> NodeExt<'a> for Node<'a> {
     fn is_descendant_of(&self, node: Node) -> bool {
-        self.range().start_byte >= node.range().start_byte
```

Comment:
This wasn't good enough because if node ranges are exactly the same we need to figure out which one is the ancestor of the other